examples/angular/src/app/components/destinations/destinations.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, inject, signal } from '@angular/core';
+import { DOTCMS_CLIENT_TOKEN } from '../../client-token/dotcms-client';
+import { GenericContentlet } from '../../utils';
+import { ContentletsComponent } from '../contentlets/contentlets.component';
+import { Contentlet } from '@dotcms/client/src/lib/client/content/shared/types';
+
+@Component({
+  selector: 'app-destinations',
+  standalone: true,
+  imports: [ContentletsComponent],
+  template: ` <div class="flex flex-col">
+    <h2 class="text-2xl font-bold mb-7 text-black">Popular Destinations</h2>
+    @if(!!destinations().length) {
+    <app-contentlets [contentlets]="destinations()" /> }
+  </div>`,
+})
+export class DestinationsComponent {
+  private readonly client = inject(DOTCMS_CLIENT_TOKEN);
+
+  readonly destinations = signal<Contentlet<GenericContentlet>[]>([]);
+
+  constructor() {
+    this.client.content
```

Comment:
@zJaaal This is an async method, I suggest all async methods should run in ngOnInit and avoid calling in the constructor.